modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -55,10 +55,13 @@ If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a s
 [id="understanding-self-node-remediation-remediation-template-config_{context}"]
 == Understanding the Self Node Remediation Template configuration
 
-The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR with the name `self-node-remediation-default-template` in the Self Node Remediation Operator's namespace. This CR defines the remediation strategy for the nodes.
+The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR. This CR defines the remediation strategy for the nodes.
```

Comment:
maybe..
- also creates the `SelfNodeRemediationTemplate` CRs.

Because we have 2 CRs:
openshift-operators     self-node-remediation-node-deletion-template       4d4h
openshift-operators     self-node-remediation-resource-deletion-template   4d4h